### ISSUE TITLE
Crash when dismissed

### ISSUE THREAD

**Issue by ahmadmssm** [algolia/voice-overlay-ios#16]
In ```VoiceOverlayController``` -> showRecordingScreen -> line number 112, app crash due to trying to set a de-allocated object, i fixed it by replacing ```un-owned``` with ```weak``` and making self optional -. self?, So please fix it.

**Comment by spinach** [comment]
Hello,

This is a similar issue to #4 I think. It should be `weak` as a safety check, but you're not supposed to get this error if you implement the VoiceOverlayController correctly on your end (see #4). Also feel free to send a PR for that, will gladly merge!

**Comment by ahmadmssm** [comment]
@spinach thanks for that, it works.
One more thing, could you add the following function to allow dismissing the voice overlay view ?!

```
public func dismiss() {
        self.inputViewController?.dismiss(animated: true, completion: nil)
    }
```
